src/MICore/Transports/LocalTransport.cs

```diff
@@ -19,9 +19,34 @@ public LocalTransport()
         {
         }
 
+        private bool IsValidMiDebuggerPath(string debuggerPath)
+        {
+            if (!File.Exists(debuggerPath))
+            {
+                return false;
+            }
+            else
+            {
+                // Verify the target is a file and not a directory
+                FileAttributes attr = File.GetAttributes(debuggerPath);
+                if ((attr & FileAttributes.Directory) != 0)
+                {
+                    return false;
+                }
+            }
+
+            return true;
+        }
+
         public override void InitStreams(LaunchOptions options, out StreamReader reader, out StreamWriter writer)
         {
             LocalLaunchOptions localOptions = (LocalLaunchOptions)options;
+
+            if (!this.IsValidMiDebuggerPath(localOptions.MIDebuggerPath))
+            {
+                throw new Exception(MICoreResources.Error_InvalidMiDebuggerPath);
```

Comment:
The message is "The value of miDebuggerPath is invalid", which seems pretty clear and actionable.  It's the same message we already use in this case for the LocalLinuxTransport.